lib/liquid/combinators/general.ex

```diff
@@ -304,16 +322,74 @@ defmodule Liquid.Combinators.General do
     parsec(:ignore_whitespaces)
     |> ignore(string(@start_filter))
     |> parsec(:ignore_whitespaces)
-    |> repeat_until(utf8_char([]), [
-      string(@start_filter),
-      string(@end_variable),
-      string(":"),
-      string(" ")
-    ])
+    |> utf8_string([not: 58, not: 124..125, not: 32], min: 1)
```

Comment:
Could you make use of module attributes instead of these integer literals?

Reply:
Done